tests/data/bundles/test_quandl.py

```diff
@@ -2,78 +2,60 @@
 
 import numpy as np
 import pandas as pd
-from toolz import merge
 import toolz.curried.operator as op
 
 from zipline import get_calendar
 from zipline.data.bundles import ingest, load, bundles
 from zipline.data.bundles.quandl import (
-    format_wiki_url,
     format_metadata_url,
+    load_data_table
 )
 from zipline.lib.adjustment import Float64Multiply
 from zipline.testing import (
     test_resource_path,
     tmp_dir,
     patch_read_csv,
 )
-from zipline.testing.fixtures import ZiplineTestCase
+from zipline.testing.fixtures import (
+    ZiplineTestCase,
+    WithResponses
```

Comment:
Not a blocker, but just to give a style recommendation, you might include a trailing comma on the final item in a list (such as multiple imports) so that the next person to add an element to the list doesn't see a change on this line as well.

Reply:
Thanks for the tip. I had noticed this in other places, but didn't know what the purpose of it was.